docs/content-services/services/category-tree-datasource.service.md

```diff
@@ -13,11 +13,12 @@ Datasource service for category tree.
 
 ### Methods
 
--   **getSubNodes**(parentNodeId: `string`, skipCount?: `number`, maxItems?: `number`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`TreeResponse<CategoryNode>`](../../../lib/content-services/src/lib/tree/models/tree-response.interface.ts)`>`<br/>
+-   **getSubNodes**(parentNodeId: `string`, skipCount?: `number`, maxItems?: `number`, name?: `string`): [`Observable`](http://reactivex.io/documentation/observable.html)`<`[`TreeResponse<CategoryNode>`](../../../lib/content-services/src/lib/tree/models/tree-response.interface.ts)`>`<br/>
```

Comment:
Do we want to display search results as a tree?

Reply:
it does not display it as tree in reality. Please check PR for applications, there is screenshot. It is tree with 0 level so in reality it will be always flat list. I did not see sense to use separated component just to render flat list if I can have tree list with 0 level which is still flat list :)